browser/brave_profile_prefs.cc

```diff
@@ -18,6 +18,7 @@
 #include "brave/components/brave_wallet/browser/buildflags/buildflags.h"
 #include "brave/components/brave_wayback_machine/buildflags.h"
 #include "brave/components/brave_webtorrent/browser/buildflags/buildflags.h"
+#include "brave/components/moonpay/buildflags/buildflags.h"
```

Comment:
This buildflags should be in `brave/components/moonpay/browser/buildflags/buildflags.h`

Reply:
moved